app/controllers/task_controller.rb

```diff
@@ -16,11 +16,38 @@ def new
   end
 
   def create
+    @person = Person.all
     @task = Task.new(create_params[:task])
     @task.save
     redirect_to root_url
   end
 
+  def edit
+    @task = Task.find(params[:id])
+    render :update
+  end
+
+  def update
+    @task = Task.find(params[:id])
+    @task.update(create_params[:task])
+    @task.save
```

Comment:
The ActiveRecord update method will inherently save the changes so you don't need to add this explicitly.